bin/edge_cases/knapsack_pro_queue_rspec_record_first_run

```diff
@@ -27,6 +27,13 @@ BRANCH_NAME=fake-branch
 #export KNAPSACK_PRO_RSPEC_SPLIT_BY_TEST_EXAMPLES=true
 #export KNAPSACK_PRO_SLOW_TEST_FILE_PATTERN="spec/**{,/*/**}/*_spec.rb"
 
+# In order to mock the Queue API responses, you need to change source code of the knapsack_pro gem:
```

Comment:
The purpose of this file was to help developer (me) to play with the knapsack_pro gem and a Rails app at the same time, so that I could see what will happen on the output, I could use a debugger etc.

We have isolated integration tests in PR https://github.com/KnapsackPro/knapsack_pro-ruby/pull/237
The isolated integration tests are already covering all edge cases (except real API calls). 

This bin script is in `bin/edge_cases` directory. It is not executed as part of `bin/knapsack_pro_all` bin script ([see](https://github.com/KnapsackPro/rails-app-with-knapsack_pro/blob/master/bin/knapsack_pro_all.rb)).